Neos.Flow/Classes/Cache/CacheFactory.php

```diff
@@ -131,8 +132,8 @@ protected function instantiateCache($cacheIdentifier, $cacheObjectName, $backend
      * @param string $backendObjectName
      * @param array $backendOptions
      * @param boolean $persistent
-     * @return FlowAbstractBackend|Backend\BackendInterface
-     * @throws Exception\InvalidBackendException
+     * @return FlowAbstractBackend|BackendInterface
+     * @throws InvalidBackendException
```

Comment:
Is the `InvalidBackendException` imported? I fail to see the corresponding `use` statement